sda-commons-server-key-mgmt/src/test/resources/mappings/mapping.yaml

```diff
@@ -11,11 +11,24 @@ mapping:
 name: SALUTATION
 mapping:
   apiToImplBidirectional:
-    - api: "MRS"
-      impl: "1"
     - api: "MR"
       impl: "0"
+    - api: "MRS"
+      impl: "1"
   implToApi:
-    - impl: "2"
-      api: "MRS"
-
+    - api: "MRS"
+      impl: "2"
+---
+name: COUNTRY
+mapping:
+  apiDefault: "DE"
```

Comment:
This is different from what I had expected. I expected that placeholders would be configured only once in config.yml. Although having default values is a good idea, placeholders and default values may not be the same thing. Maybe we were all talking about different things here? I think we need to dicuss this with Stefan on monday.